src/include/sof/lib/dai.h

```diff
@@ -23,6 +23,7 @@
 #include <errno.h>
 #include <stddef.h>
 #include <stdint.h>
+#include <sof/audio/component.h> /* COMP_TRIGGER_* macros */
```

Comment:
Nothing else added in this header.

Reply:
Leftover from some old traces, will remove.

Reply:
Actually they were in use, moved the include to esai.c.